pages/news/2020-08-20-Meter-Detection.html

```diff
@@ -0,0 +1,41 @@
+title: Experimental Meter Detection Code
```

Comment:
The title is not really catchy.

Maybe something like:
```suggestion
title: GSoC 2020: Adding experimental Meter Detection
```